packages/flutter_localizations/lib/src/l10n/material_af.arb

```diff
@@ -25,6 +25,7 @@
   "continueButtonLabel": "GAAN VOORT",
   "copyButtonLabel": "Kopieer",
   "cutButtonLabel": "Knip",
+  "scanTextButtonLabel": "Skena umbhalo",
```

Comment:
Yes. This is for Material style button label. Although android doesn't have that. But on iOS, if the `ThemeData` 's platform set to android (but current real platform is iOS)like below.

``` dart
return MaterialApp(
      checkerboardRasterCacheImages: true,
      title: 'Flutter Demo',
      theme: ThemeData(
        platform: TargetPlatform.android,
      ),
      home: const TextFieldPage(),
);
```
It can still works fine.
<img src="https://user-images.githubusercontent.com/49340347/210520867-74ab8335-f51d-4cf6-ae3f-e3d6a60bc138.jpg" width=200 />

Reply:
I thought we had a guide for how to do this kind of stuff, but I can't find it.  I posted on Discord looking for help: https://discord.com/channels/608014603317936148/608021234516754444/1060266638513033216

Reply:
Sounds good, this is the guide I was thinking of, which you have probably already seen:

But do you also need to add this localization to Cupertino so that you can use it in CupertinoTextSelectionToolbarButton.getButtonLabel?

Reply:
I think CupertinoTextSelectionToolbarButton doesn't need that string because we will use icon for that button. So I just return `''` ,just the same as `ContextMenuButtonType.delete` O(∩_∩)O